### DISCUSSION TITLE
need json schema structure for input data

### DISCUSSION THREAD

**Discussion by mavijayk** [json-schema-org/community#541] [Q&A]
Hi All, below is the json input file structure I'm receiving daily,
 **ETL** : Informatica Cloud Intelligence
**Source** : Json event files

**sample data**
[
{
	"apple-2023-11-21": [
		{
			"name": "apple",
			"type": "fruit",
			"taste": "sweet"
		}
	],
	"potato-2023-11-21": [
		{
			"name": "potato",
			"type": "veggie",
			"taste": "sour"
		}
	]
}
]

**schema I have generated**

{
	"$id": "https://example.com/object1700667140.json", 
	"type": "array",
	"default": [],
	"items":{
		"type": "object",
		"properties": {
			"apple-2023-11-21": {
				"type": "array",
				"default": [],
				"items":{
					"type": "object",
					"properties": {
						"name": {
							"type": "string",
							"default": "",
							"pattern": "^.*$"
						},
						"type": {
							"type": "string",
							"default": "",
							"pattern": "^.*$"
						},
						"taste": {
							"type": "string",
							"default": "",
							"pattern": "^.*$"
				}
		}}},
		"potato-2023-11-21": {
				"type": "array",
				"default": [],
				"items":{
					"type": "object",
					"properties": {
						"name": {
							"type": "string",
							"default": "",
							"pattern": "^.*$"
						},
						"type": {
							"type": "string",
							"default": "",
							"pattern": "^.*$"
						},
						"taste": {
							"type": "string",
							"default": "",
							"pattern": "^.*$"
						}	}	}	}	}}}

Above tile name is dynamic value changes in a pattern "fruitname-yyyy-mm-dd" now problem here is, Informatica NOT able to parse the json file which is coming in pattern like above. I tired many options/properties etc. but didn't work. kindly do the needful and appreciate your help.

**Regards**,
Vijay Kumar

**Comment by benjagm** [comment]
Closed as Duplicated: #540
